example/client/main.cpp

```diff
@@ -0,0 +1,225 @@
+//
+// Copyright (c) 2024 Mohammad Nejati
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/cppalliance/http_io
+//
+
+#include <boost/asio.hpp>
+#include <boost/asio/ssl.hpp>
+#include <boost/http_io.hpp>
+#include <boost/http_proto.hpp>
+#include <boost/url.hpp>
+
+#include <cstdlib>
+#include <functional>
+#include <iostream>
+
+namespace asio       = boost::asio;
+namespace core       = boost::core;
+namespace http_proto = boost::http_proto;
+namespace io         = boost::http_io;
+namespace ssl        = boost::asio::ssl;
+namespace urls       = boost::urls;
+using tcp            = boost::asio::ip::tcp;
+using namespace std::placeholders;
```

Comment:
Doing all the aliases is something I tried out and later realized I do not like.

Reply:
I kept the namespace aliases because they reduce some noise from the examples and make them easier to read. Do you think we should remove them?